.github/workflows/npm-ecdsa.yml

```
@@ -0,0 +1,55 @@
+name: NPM ECDSA
+
+on:
+  push:
+    branches:
+      - main
+      - npm-ecdsa # TODO: remove before merging to main
+    paths:
+      - "solidity/ecdsa/contracts**"
```

Shouldn't it be `contracts/**`?
```suggestion
      - "solidity/ecdsa/contracts/**"
```